inginious/frontend/pages/social.py

```diff
@@ -43,14 +43,6 @@ def process_callback(self, auth_id):
         if user and auth_storage.get("method", "") == "signin":
```

Comment:
There should be no more need to store the method in `auth_storage` and there probably is some additional code that can be removed.